app/views/patient_mailer/assessment_email.html.erb

```diff
@@ -8,6 +8,12 @@
 
     <p><%= t('assessments.email.reminder.thank-you', locale: @lang) %></p>
 
+    <% if Jurisdiction.find_by_id(patient[:patient].jurisdiction_id).contact_info[:email] != '' || Jurisdiction.find_by_id(patient[:patient].jurisdiction_id).contact_info[:phone] != '' || Jurisdiction.find_by_id(patient[:patient].jurisdiction_id).contact_info[:webpage] != '' %>
```

Comment:
instead of performing this query multiple times in the view, we can define it in the mailer (similar to how `@lang` is defined there)
```suggestion
    <% if @contact_info[:email].present? || @contact_info[:phone].present? || @contact_info[:webpage].present? %>
```

Reply:
resolved in https://github.com/SaraAlert/SaraAlert/pull/925/commits/84b7c09c4bd54b625ebe69dd11114ffefacdf592